src/main/resources/db/migration/V2__create_concert_venues.sql

```diff
@@ -5,7 +5,7 @@ CREATE TABLE concert_venues (
   city VARCHAR(255) NOT NULL,
   state VARCHAR(2) NOT NULL,
   zip_code VARCHAR(10) NOT NULL,
-  phone_number VARCHAR(10) NOT NULL,
+  phone_number VARCHAR(14) NOT NULL,
```

Comment:
great :) 

can we rename this migration to `V1` or are we waiting on a `V1` from another pair?